CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.13.0)
```

Comment:
Thinking more about this, consider setting `3.13.4`. It fixes few regressions and is the version tested.
See https://cmake.org/cmake/help/v3.13/release/3.13.html#id1

Reply:
Update: I will take care of the change when integrating.

Reply:
I can open PR's for the remotes and update the hashes tomorrow